relayer-cli/src/components.rs

```diff
@@ -1,10 +1,14 @@
 use abscissa_core::{Component, FrameworkError, FrameworkErrorKind};
 use tracing_subscriber::{filter::EnvFilter, util::SubscriberInitExt, FmtSubscriber};
 
-use ibc_relayer::config::GlobalConfig;
+use ibc_relayer::config::{GlobalConfig, LogLevel};
 
 use crate::config::Error;
 
+/// The name of the environment variable through which one can override
+/// the tracing filter built in [`build_tracing_filter`].
+const HERMES_LOG_VAR: &str = "RUST_LOG";
```

Comment:
I recall the integration tests framework making use of `RUST_LOG`, eg in `cargo test`. Would just like to flag that -- is there any interference between this approach and the pre-existing support for `RUST_LOG` in cargo?

Reply:
The logs from the integration tests look fine to me: https://github.com/informalsystems/ibc-rs/runs/5272869928?check_suite_focus=true (modulo some doc tests failures, unrelated to this PR and best fixed elsewhere)

Reply:
By the way, just to clarify, `cargo test` does not care itself about `RUST_LOG` but the integration tests themselves do, cf. https://github.com/informalsystems/ibc-rs/blob/8329d856844df30651f68a25ca5c5db4addbaa2b/tools/integration-test/src/bootstrap/single.rs#L75

Reply:
What the test suite does is look at the `RUST_LOG` env variable, and then sets the `log_level` property in `config.toml` to that. If the test suite is only ever run with `RUST_LOG=info/trace/debug/error/warn` then everything should work as expected with no change in the output.

Reply:
> By the way, just to clarify, `cargo test` does not care itself about `RUST_LOG` but the integration tests themselves do, cf.

Thanks for clarifying that! This part wasn't obvious to me.